examples/look/look/images.py

```diff
@@ -8,18 +8,11 @@
 
 
 class Resource:
-
     def __init__(self, image_store):
         self._image_store = image_store
 
     def on_get(self, req, resp):
-        doc = {
-            'images': [
-                {
-                    'href': '/images/1eaf6ef1-7f2d-4ecc-a8d5-6e8adba7cc0e.png'
-                }
-            ]
-        }
+        doc = {'images': [{'href': '/images/1eaf6ef1-7f2d-4ecc-a8d5-6e8adba7cc0e.png'}]}
```

Comment:
NOTE: this could be improved a bit: placing a comma `,` at the end of the end of a definition makes black use new lines, so we can force expanded formatting for dict/string/argument lists, so that the formatted version is closer to the existing one. It would become:

```py
        doc = {
            'images': [
                {
                    'href': '/images/1eaf6ef1-7f2d-4ecc-a8d5-6e8adba7cc0e.png',
                },
            ],
        }
```

Reply:
IIRC this look stuff used to be copypasted, so we'll need to sync, or use "literal include" like I did in the ASGI version (thanks @CaselIT).

Reply:
ok let's add some points to do